api/admin/service.go

```diff
@@ -128,6 +130,29 @@ func (service *Admin) AliasChain(_ *http.Request, args *AliasChainArgs, reply *a
 	return service.httpServer.AddAliasesWithReadLock("bc/"+chainID.String(), "bc/"+args.Alias)
 }
 
+// GetChainAliasesArgs are the arguments for calling GetChainAliases
+type GetChainAliasesArgs struct {
+	Chain string `json:"chain"`
+}
+
+// GetChainAliasesReply are the aliases of the given chain
+type GetChainAliasesReply struct {
+	Aliases []string `json:"aliases"`
+}
+
+// GetChainAliases returns the aliases of the chain
+func (service *Admin) GetChainAliases(r *http.Request, args *GetChainAliasesArgs, reply *GetChainAliasesReply) error {
+	service.log.Info("Admin: GetChainAliases called with Chain: %s", args.Chain)
+
+	ID, err := ids.FromString(args.Chain)
```

Comment:
It looks like it was already like this... but since we are here can we clean up the `ID` variable? The capitalization isn't in line with the rest of the repo. Perhaps we could change the name to `chainID` or `id`?

Reply:
Golint wants `chainId` to be `chainID` so I renamed as `id`